Update iis.md (#5932)
umbraco/UmbracoDocs: 13/umbraco-cms/fundamentals/setup/server-setup/iis.md

## Configuring IIS for .NET

> * Install the [ASP.NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/) and download the **Hosting Bundle**. Ensure you download the correct version for your specific version of Umbraco.

Reviewer: ASP.NET is from below v9, and from v9 is just .NET. We also have the correct .NET version that needs to be installed in the requirements.
```suggestion
* Install the [NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/) and download the **Hosting Bundle**. Ensure you download the correct .NET version as per the [requirements](../requirements.md).
```

> * Install the [ASP.NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/) and download the **Hosting Bundle**. Ensure you download the correct .NET version as per the [requirements](../requirements.md).

Reviewer: ```suggestion
* Install the [.NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/) and download the **Hosting Bundle**. Ensure you download the correct .NET version as per the [requirements](../requirements.md).
```
